src/Core_Command.php

```diff
@@ -242,11 +244,16 @@ public function download( $args, $assoc_args ) {
 		if ( $cache_file ) {
```

Comment:
The `$extract` should only be checked once.
Otherwise, we have the source of truth of the default argument in two places, which is a potential for bugs later down the road.
```suggestion
		$extract   = (bool) Utils\get_flag_value( $assoc_args, 'extract', true );

		if ( $cache_file ) {
```